src/features/targeting/constants.js

```diff
@@ -0,0 +1,9 @@
+// @flow
+const adhoc: 'adhoc' = 'adhoc';
+const banner: 'banner' = 'banner';
+const modal: 'modal' = 'modal';
+const popout: 'popout' = 'popout';
+
```

Comment:
empty line needed?

Reply:
in this case, I think empty line serve as a separation from individual constants to the object. It might be acceptable to put empty line there. I looked through existing code in box-ui-elements and found they use capital case for constants naming and I will change to use capital case.
Thanks for review.